src/internal/operators/onErrorResumeNext.ts

```diff
@@ -83,7 +83,11 @@ export function onErrorResumeNext<T, A extends readonly unknown[]>(
 export function onErrorResumeNext<T, A extends readonly unknown[]>(
   ...sources: [[...ObservableInputTuple<A>]] | [...ObservableInputTuple<A>]
 ): OperatorFunction<T, T | A[number]> {
-  const nextSources = argsOrArgArray(sources);
+  // For some reason, TS 4.1 RC gets the inference wrong here and infers the
+  // result to be `A[number][]` - completely dropping the ObservableInput part
+  // of the type. This makes no sense whatsoever. As a workaround, the type is
+  // asserted explicitly.
+  const nextSources = (argsOrArgArray(sources) as unknown) as [[...ObservableInputTuple<A>]];
```

Comment:
nit: Can this also be `as [ObservableInputTuple<A>]`?

Reply:
Done. I think it should have been `ObservableInputTuple<A>`, actually.